src/main/java/seedu/address/model/task/Deadline.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.task;
+
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Task's deadline in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidEmail(String)}
+ */
+public class Deadline {
+
+    public static final String MESSAGE_DEADLINE_CONSTRAINTS =
```

Comment:
Is `public` the best access modifier for `MESSAGE_DEADLINE_CONSTRAINTS` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.